instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-wildfly-testing/jaxrs-2.0-wildfly-testing.gradle

```diff
@@ -0,0 +1,69 @@
+ext.skipPublish = true
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+configurations {
+  testServer
+}
+
+dependencies {
+  api "javax:javaee-api:7.0"
+
+  def arquillianVersion = '1.4.0.Final'
+  testImplementation "org.jboss.arquillian.junit:arquillian-junit-container:${arquillianVersion}"
+  testImplementation "org.jboss.arquillian.protocol:arquillian-protocol-servlet:${arquillianVersion}"
+  testImplementation "org.wildfly.arquillian:wildfly-arquillian-container-embedded:2.2.0.Final"
+  testImplementation 'org.jboss.arquillian.spock:arquillian-spock-container:1.0.0.CR1'
+  testImplementation "org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-gradle-depchain:3.1.3"
+  testImplementation "org.glassfish.jersey.core:jersey-client:2.8"
+
+  testInstrumentation project(':instrumentation:servlet:servlet-3.0:javaagent')
+  testInstrumentation project(':instrumentation:jaxrs:jaxrs-2.0:jaxrs-2.0-common:javaagent')
+
+  // wildfly version used to run tests
+  testServer "org.wildfly:wildfly-dist:18.0.0.Final@zip"
+}
+
+// extract wildfly dist, path is used from arquillian.xml
+task setupServer(type: Copy) {
+  from zipTree(configurations.testServer.singleFile)
+  into file('build/server/')
+}
+
+// logback-classic contains /META-INF/services/javax.servlet.ServletContainerInitializer
```

Comment:
This looks familiar - @trask I see to recall we had a better way around this?

Reply:
yes, it does look familiar 😄

https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1635#discussion_r523703382

if this becomes a common problem, we could have a test-logging module that does the exclusion, and instrumentation.gradle can depend on the test-logging module, instead of pulling in logback directly

Reply:
Or switch to log4j :)

Reply:
ya, i'm not even sure what we are logging directly from test code (separate from the agent's internal logging)